pm/recipient_test.go

```diff
@@ -0,0 +1,436 @@
+package pm
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
+	"math/big"
+	"testing"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+func TestReceiveTicket(t *testing.T) {
+	sender := ethcommon.HexToAddress("A69cdA26600c155cF2c150964Bdb5371ac3f606F")
```

Comment:
Is this specific address important, or would any random address work here?

If a random address is just as good, I'd like to suggest using the approach I took [here](https://github.com/livepeer/go-livepeer/pull/651/files#diff-0800fac1c4a865f1fd74c8aeebcd8e60R231), basically generating a random address every time the test runs.

The advantage is a more readable test - because I don't have to wonder about the specific significance of this address.

wdyt?